SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1139,7 +1141,34 @@ int32 USpatialNetDriver::ServerReplicateActors_PrepConnections(const float Delta
 	return bFoundReadyConnection ? NumClientsToTick : 0;
 }
 
-int32 USpatialNetDriver::ServerReplicateActors_PrioritizeActors(UNetConnection* InConnection, const TArray<FNetViewer>& ConnectionViewers, const TArray<FNetworkObjectInfo*> ConsiderList, const bool bCPUSaturated, FActorPriority*& OutPriorityList, FActorPriority**& OutPriorityActors)
+struct FCompareActorPriorityAndMigration
```

Comment:
Can we move this struct into the Migrationhandler and have a API method on the handler to sort Actors - this is so as much of the migration logic as possible is in the handler.

Reply:
Sorting actors is something the NetDriver wants to do. The migration handler would then need to know about FActorPriority, which in terms of object design does not make a lot of sense.

Reply:
I disagree, the priority purely depends on migration currently, and I think there's benefit in not polluting the NetDriver with new load balancing stuff